error: use of `write!(stdout(), ...).unwrap()`
  --> $DIR/explicit_write.rs:17:9
   |
LL |         write!(std::io::stdout(), "test").unwrap();
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `print!("test")`
   |
   = note: `-D clippy::explicit-write` implied by `-D warnings`

error: use of `write!(stderr(), ...).unwrap()`
  --> $DIR/explicit_write.rs:18:9
   |
LL |         write!(std::io::stderr(), "test").unwrap();
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `eprint!("test")`

error: use of `writeln!(stdout(), ...).unwrap()`
  --> $DIR/explicit_write.rs:19:9
   |
LL |         writeln!(std::io::stdout(), "test").unwrap();
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `println!("test")`

error: use of `writeln!(stderr(), ...).unwrap()`
  --> $DIR/explicit_write.rs:20:9
   |
LL |         writeln!(std::io::stderr(), "test").unwrap();
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `eprintln!("test")`

error: use of `stdout().write_fmt(...).unwrap()`
  --> $DIR/explicit_write.rs:21:9
   |
LL |         std::io::stdout().write_fmt(format_args!("test")).unwrap();
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `print!("test")`

error: use of `stderr().write_fmt(...).unwrap()`
  --> $DIR/explicit_write.rs:22:9
   |
LL |         std::io::stderr().write_fmt(format_args!("test")).unwrap();
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `eprint!("test")`

error: use of `writeln!(stdout(), ...).unwrap()`
  --> $DIR/explicit_write.rs:25:9
   |
LL |         writeln!(std::io::stdout(), "test/ntest").unwrap();
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `println!("test/ntest")`

error: use of `writeln!(stderr(), ...).unwrap()`
  --> $DIR/explicit_write.rs:26:9
   |
LL |         writeln!(std::io::stderr(), "test/ntest").unwrap();
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `eprintln!("test/ntest")`

error: aborting due to 8 previous errors

